launcher/ui/dialogs/AboutDialog.ui

```diff
@@ -80,7 +80,7 @@
       </font>
      </property>
      <property name="text">
-      <string notr="true">PolyMC</string>
+      <string notr="true">Launcher</string>
```

Comment:
This will be replaced programmatically anyway, so this could also say `pee-pee poo-poo`

Reply:
Ok